download.php

```diff
@@ -1,28 +1,59 @@
 <div class="row">
-	<div class="col-sm-4">
-		<h3>Source</h3>
-		<p>Fork us at <a href="https://github.com/namecoin/namecoin">Github</a>.</p>
-	</div>
-	<div class="col-sm-4">
-		<h3>Windows</h3>
-		<p>
-			Download <a href="https://namecoin.info/files/Namecoin_v0.3.80_setup.exe">v0.3.80 installer</a>.<br>
-			<a target="_blank" href="https://forum.namecoin.info/viewtopic.php?f=8&t=2123">Release information and signatures</a>.
-		</p>
-	</div>
-	<div class="col-sm-4">
-		<h3>Mac OS X</h3>
-		<p>
-			Download <a href="https://namecoin.info/files/Namecoin-Qt.app-0.3.80-a00c33d.zip">v0.3.80 Namecoin-Qt</a>.<br>
-			<a target="_blank" href="https://forum.namecoin.info/viewtopic.php?f=8&t=2235">Release information</a>.
-		</p>
-	</div>
-	<div class="col-sm-4">
-		<h3>Linux Packages</h3>
-		<p>
-			Linux packages provided by OBS.<br>
-			<a href="https://software.opensuse.org/download.html?project=home%3Ap_conrad%3Acoins&amp;package=namecoin">Command line client (namecoind)</a>.<br>
-			<a href="https://software.opensuse.org/download.html?project=home%3Ap_conrad%3Acoins&amp;package=namecoin-gui">Graphical interface (Namecoin-Qt)</a>.
-		</p>
-	</div>
-</div>
\ No newline at end of file
+    <h2>Namecoin-Qt Client</h2>
+        <p>
+            Client for normal use (but not for mining or development). Always wait for six confirmations.
+            <a href="https://github.com/namecoin/namecoin">Source on Github</a>.
+        </p>
+        <div class="col-sm-4">
+            <h3>Windows</h3>
+            <p>
+                Download <a href="https://namecoin.info/files/Namecoin_v0.3.80_setup.exe">v0.3.80 installer</a>.<br>
+                <a target="_blank" href="https://forum.namecoin.info/viewtopic.php?f=8&t=2123">Release information and signatures</a>.
+            </p>
+        </div>
+        <div class="col-sm-4">
+            <h3>Linux Packages</h3>
+            <p>
+                Linux packages provided by OBS.<br>
+                <a href="https://software.opensuse.org/download.html?project=home%3Ap_conrad%3Acoins&amp;package=namecoin">Command line client (namecoind)</a>.<br>
+                <a href="https://software.opensuse.org/download.html?project=home%3Ap_conrad%3Acoins&amp;package=namecoin-gui">Graphical interface (Namecoin-Qt)</a>.
+            </p>
+        </div>
+        <div class="col-sm-4">
+            <h3>Mac OS X</h3>
+            <p>
+                Download <a href="https://namecoin.info/files/Namecoin-Qt.app-0.3.80-a00c33d.zip">v0.3.80 Namecoin-Qt</a>.<br>
+                <a target="_blank" href="https://forum.namecoin.info/viewtopic.php?f=8&t=2235">Release information</a>.
+            </p>
+        </div>
+</div>
+<div class="row">
+    <h2>NMControl Middleware</h2>
+        <p>
+            NMControl connects your browser/application to the client (or soon to an API). It allows you, for example, to browse .bit domains.
```

Comment:
Thanks for elaborating. You convinced me with SPV alternatives. We still could have said something like "we are working on lighter alternatives" but I think it might sound like running the client is overly difficult so I will just remove this altogether.